UF_OAF_Analysis.R

```diff
@@ -16,62 +16,68 @@ rm(list=ls())
 ######################################################
 #Step 1: load the CSV file and save as dataframes
 
-OAF<-read.csv("UFOAFData_EB.csv", dec=".", header = TRUE, sep = ",", check.names=FALSE, stringsAsFactors=FALSE) #need to save strings so tha you can error correct
+OAF <- read.csv("UFOAFData_EB.csv", dec=".", header = TRUE, sep = ",", check.names=FALSE, stringsAsFactors=FALSE) #need to save strings so tha you can error correct (ERS: this is now default behavior as of a recent R version)
```

Comment:
actually, I prefer to do this using read_csv going forward, so while the formatting is better I'm not acvcepting until that switch is made